from_mwt_ds/DataScience/vw_executor/tests/data/artifacts/pred_csoaa_ldf.txt

```diff
@@ -0,0 +1,10 @@
+1
+0
+0
+
+0
+2
+
+0
+3
+
```

Comment:
just so I understand, the output would be:

{index: 0, label: 1}
{index: 1, label: 2}
{index: 1, label: 3}

correct?
(might be a good idea to add this expectation to the unit test as well?)
thanks for adding this!

Reply:
sorry for missing the question.
yes, that's correct